api/v1alpha1/edgedevice_types.go

```diff
@@ -44,6 +44,7 @@ type MetricsReceiverConfiguration struct {
 	RequestNumSamples int64  `json:"requestNumSamples,omitempty"`
 	TimeoutSeconds    int64  `json:"timeoutSeconds,omitempty"`
 	URL               string `json:"url,omitempty"`
+	SecretName        string `json:"secretName,omitempty"`
```

Comment:
Are there plans to store any other certs in that secret besides CA? 
If not, please change the name to something more self-describing ('CASecretName`, `CASecret`, etc).